src/components/layout/SidebarLinks.tsx

```diff
@@ -25,9 +27,33 @@ const renderLink = (
 ) => {
   const { isChild, onLinkClick } = context ?? {}
 
+  function renderIcon(): ReactNode {
```

Comment:
Sidebar icons for the Acme theme.

<img src="https://github.com/user-attachments/assets/c296bd95-a09c-4d81-93af-b833cce19b39" width="280px">